atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -126,6 +131,28 @@ def generate_auto_labels(self, auto_labels):
         docker_info = self.tasker.get_info()
         generated['com.redhat.build-host'] = docker_info['Name']
 
+        info_url = self.info_url_format
+        labels = Labels(self.labels)
+        find_subs = re.finditer('{([^}]+)}', info_url)
```

Comment:
Note that if setting 'url' needs to happen in a separate plugin to make this implementation easier, let's do that. We certainly don't have to have *every* label adjustment done in this one plugin -- we already adjust 'release' separately in bump_release, for example.

Let's keep it simple.